### DISCUSSION TITLE
Why vscode doesn't suggest anything in .vue file for nuxt modules

### DISCUSSION THREAD

**Discussion by huuhait** [nuxt/nuxt#17414] [Questions]
I've to try hover typescript line
but nothing happen
![image](https://user-images.githubusercontent.com/40519768/174456165-79d93bda-80df-41ff-91ca-61ea75065b95.png)

**Comment by Flanker72** [comment]
Please, check that you installed [Volar extension](https://marketplace.visualstudio.com/items?itemName=johnsoncodehk.volar) and [TypeScript Vue Plugin](https://marketplace.visualstudio.com/items?itemName=johnsoncodehk.vscode-typescript-vue-plugin).

If you have issues with Volar/TypeScript - you may use `Ctrl+Shift+P` on Windows or `Cmd + Shift + P` on Mac and run "`Volar: Restart Vue server`" or "`TypeScript: Restart TS server`" to reinitialize them without restarting IDE.

Also, please, consider, that Nuxt requires type generation before they can be referenced from IDE. You may run `npm run dev` to start the development server (types and references will be generated on the fly), or run `npx nuxi prepare` to just generate typings.
That's because Nuxt default configuration for TypeScript and Vue references the generated files within the `.nuxt` folder, which is created after running one of the above commands.

**Reply by huuhait** [reply]
I've installed this plugins
working good in vue project and nuxt project
but in nuxt module project src folder volar not working
playground folder working good

**Reply by huuhait** [reply]
I've tested move components to outside of runtime folder
volar working good

any idea?
![image](https://user-images.githubusercontent.com/40519768/174494386-f59fba48-203d-4488-bfd0-1ead9c3c6a14.png)
![image](https://user-images.githubusercontent.com/40519768/174494392-58a4c3c6-08eb-4a57-b32a-dba3a6e95706.png)
